packages/terra-i18n/src/translations/de.js

```diff
@@ -0,0 +1,31 @@
+import { addLocaleData } from 'react-intl';
+import localeData from 'react-intl/locale-data/de';
+
+addLocaleData(localeData);
+
+const messages = {
+  'Terra.ajax.error': 'Inhalt konnte nicht geladen werden.',
```

Comment:
As we already test all 8 locales in application level, I think it's nice to have at this point. We can discuss in today's meetings on that.